### DISCUSSION TITLE
2022/02/12/twelveth-post

### DISCUSSION THREAD

**Discussion by giscus** [wookdev/munged-org#8] [Blog Comments]
# 2022/02/12/twelveth-post

A second test post front matter excerpt

http://localhost:4000/2022/02/12/twelveth-post.html

**Comment by wookdev** [comment]
# OMG this is a comment for twelfth post!!!

This line added via editing the comment.

**Comment by wookdev** [comment]
# Editing...

Can't edit posts.

At least, not here.

**Comment by wookdev** [comment]
first line is not a header tag
